owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/security/PassCodeViewModelTest.kt

```diff
@@ -146,4 +166,19 @@ class PassCodeViewModelTest : ViewModelTest() {
         }
     }
 
+    @Test
+    fun `get number of passcode digits - ok`() {
+        val numberDigits = 4
```

Comment:
Same here

Reply:
Well in this case I don't think it's worth it to move this 4 to the owncloudTestUtil module, but I made some changes in the rest of the tests which used some other examples :)